src/track/trackmetadata.h

```diff
@@ -194,6 +202,7 @@ class TrackMetadata {
     int m_channels;
     int m_duration; // seconds
     int m_sampleRate; // Hz
+    int m_iRating;
```

Comment:
TrackMetadata stores tags from various file formats in a portable way, both for reading and writing. I don't recommend to implement it in a way that just works for some special cases and loses information when writing tags back to files as Daniel already mentioned.

The int-rating in Mixxx is just the number of stars displayed on the UI. A portable solution needs to provide a bidirectional mapping between the tags from various file formats and the internal representation.